src/lib.rs

```diff
@@ -23,25 +23,25 @@ pub fn eval_beat(cmds: &[Cmd], t: i32) -> Result<i32, ()> {
             Cmd::Add => {
                 let b = stack.pop().ok_or(())?;
                 let a = stack.pop().ok_or(())?;
-                stack.push(a + b);
+                stack.push(a.wrapping_add(b));
             }
             Cmd::Sub => {
                 let b = stack.pop().ok_or(())?;
                 let a = stack.pop().ok_or(())?;
-                stack.push(a - b);
+                stack.push(a.wrapping_sub(b));
             }
             Cmd::Mul => {
                 let b = stack.pop().ok_or(())?;
                 let a = stack.pop().ok_or(())?;
-                stack.push(a * b);
+                stack.push(a.wrapping_mul(b));
             }
             Cmd::Div => {
                 let b = stack.pop().ok_or(())?;
                 let a = stack.pop().ok_or(())?;
                 if b == 0 {
                     stack.push(0);
                 } else {
-                    stack.push(a / b);
+                    stack.push(a.wrapping_div(b));
```

Comment:
This is good for symmetry, and I did it also, but I'm suspicious that this probably is unnecessary/does nothing.